db/jobs.go

```diff
@@ -60,12 +60,14 @@ type JobFilter struct {
 }
 
 func (f *JobFilter) Query(driver string) (string, []interface{}, error) {
-	if f.TenantID == "" {
-		return "", nil, fmt.Errorf("tenant id is required for job retrieval")
-	}
-	wheres := []string{"j.tenant_uuid = ?"}
-	args := []interface{}{f.TenantID}
-
+	//if f.TenantID == "" {
+	//	return "", nil, fmt.Errorf("tenant id is required for job retrieval")
+	//}
+	//wheres := []string{"j.tenant_uuid = ?"}
+	//args := []interface{}{f.TenantID}
+	wheres := []string{"j.uuid = j.uuid"}
+
+	var args []interface{}
```

Comment:
Revert this; it gets fixed in another branch and you don't want those merge conflicts, trust me.